src/Tickets/Commerce/Hooks.php

```diff
@@ -324,16 +324,14 @@ public function do_checkout_parse_request() {
 	/**
 	 * Backwards compatibility to update stock after deletion of Ticket.
 	 *
-	 * @todo  Determine if this is still required.
-	 *
 	 * @since 5.1.9
 	 *
-	 * @param int $ticket_id  the attendee id being deleted.
-	 * @param int $post_id    the post or event id for the attendee.
-	 * @param int $product_id the ticket-product id in Tribe Commerce.
+	 * @since TBD Updated method to use the new hook from Attendee class.
+	 *
+	 * @param int $attendee_id the attendee id.
 	 */
-	public function update_stock_after_deletion( $ticket_id, $post_id, $product_id ) {
-		$this->container->make( Ticket::class )->update_stock_after_deletion( $ticket_id, $post_id, $product_id );
+	public function update_stock_after_attendee_deletion( $attendee_id ) {
+		$this->container->make( Ticket::class )->update_stock_after_attendee_deletion( $attendee_id );
```

Comment:
We can't just delete a method like that when it's public, move it to the end and deprecate it properly.